src/main/java/com/meilisearch/sdk/TasksHandler.java

```diff
@@ -88,11 +88,11 @@ TasksResults getTasks(String indexUid) throws MeilisearchException {
      * @throws MeilisearchException if client request causes an error
      */
     TasksResults getTasks(String indexUid, TasksQuery param) throws MeilisearchException {
-        String[] newIndexUid = new String[param.getIndexUid().length + 1];
-        if (param != null && param.getIndexUid() != null) {
-            for (int i = 0; i < param.getIndexUid().length; i++)
-                newIndexUid[i] = param.getIndexUid()[i];
-            newIndexUid[param.getIndexUid().length] = indexUid;
+        String[] newIndexUid = new String[param.getIndexUids().length + 1];
```

Comment:
If I understand this piece correctly, it gets all the `indexUids` sent in the `TaskQuery`, which can be none. And add the current index uid, correct?

Reply:
I would like to see a test where we explicitly assert for the values in this final array.